sdk/spring/README.md

```diff
@@ -15,10 +15,27 @@ cd azure-sdk-for-java
 mvn clean package -f sdk/spring/pom.xml -P dev
 ```
 
-And if you want to install jars into your local repository:
+You can use the following command to install jars into your local repository quickly:
 
 ```shell
-mvn clean install -f sdk/spring/pom.xml -P dev
+mvn clean install \
+  -f sdk/spring/ \
+  -Pdev \
+  -T 4 \
+  -o \
+  -Dcheckstyle.skip \
+  -Dcodesnippet.skip \
+  -Denforcer.skip \
+  -Djacoco.skip \
+  -Dmaven.javadoc.skip \
+  -Dmaven.test.skip \
+  -Drevapi.skip \
+  -Dspotbugs.skip
+```
+
+If you are using PowerShell, please use the following command instead:
+```shell
+mvn clean install -f sdk/spring/ -Pdev -T 4 -o "-Dcheckstyle.skip" "-Dcodesnippet.skip" "-Denforcer.skip" "-Djacoco.skip" "-Dmaven.javadoc.skip" "-Dmaven.test.skip" "-Drevapi.skip" "-Dspotbugs.skip"
```

Comment:
in powershell you can use `

Reply:
![image](https://user-images.githubusercontent.com/13167207/159192522-150fe3d5-a2ff-4deb-bff5-b78fff4e409f.png)